Podfile.lock

```diff
@@ -24,12 +24,12 @@ EXTERNAL SOURCES:
     :path: "./"
 
 SPEC CHECKSUMS:
-  MiniApp: 228a479d18ea93de55b235a0481576560fb2f909
-  Nimble: 5f8a2fb6fa343a7242dfdd9d42f7267419d464b2
+  MiniApp: 81660bb845ce64d275f9f45ccd7742d6fb45617a
+  Nimble: 3864815b4703c7ebffba875973c70e854489fbae
   Quick: 6d9559f40647bc4d510103842ef2fdd882d753e2
   RSDKUtils: 9940b99fa0494181b8ee2cfa2ac742bb0958b984
   ZIPFoundation: b1f0de4eed33e74a676f76e12559ab6b75990197
 
 PODFILE CHECKSUM: ca9d5c9f0e66d29311ef20fb6d4ac797977bd6bf
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.8.4
```

Comment:
Could you please update the Cocoapods as well

Reply:
yeah, I don't know why fastlane does not use the same my system is using